### PR TITLE
Bump NNS FE dapp to proposal-123301 / 85710a82e0c4b287b43a7c97a8e6b2f231f800e4

### PR DESCRIPTION
**Pull request by aterga**
https://github.com/dfinity/nns-dapp/releases/tag/proposal-123301

### CODE REVIEW

**Review by mraszyk — APPROVED** [review]
LGTM as long as it works (did you try?)

**Comment by mraszyk** [comment]
Makes sense. Thank you!

File: setup.sh [hunk 1]
```diff
@@ -67,7 +67,13 @@ fi
 if [ ! -z "${NNS_DAPP_RELEASE:-}" ]
 then
   mkdir -p nns-dapp/out
-  curl -L "https://github.com/dfinity/nns-dapp/releases/download/${NNS_DAPP_RELEASE}/nns-dapp_local.wasm" -o nns-dapp/out/nns-dapp.wasm
+  NNS_DAPP_WASM_PREFIX="https://github.com/dfinity/nns-dapp/releases/download/${NNS_DAPP_RELEASE}/"
+  FOUND_LOCAL_NNS_DAPP_WASM=$(curl --silent -I -L "${NNS_DAPP_WASM_PREFIX}/nns-dapp_local.wasm" | grep "^HTTP/2 200")
+  if [ ${FOUND_LOCAL_NNS_DAPP_WASM} ]; then
+    curl -L "${NNS_DAPP_WASM_PREFIX}/nns-dapp_local.wasm" -o nns-dapp/out/nns-dapp.wasm
```
Comment (mraszyk) [inline setup.sh:73]: Why ever use `nns-dapp_local.wasm` instead of `nns-dapp.wasm`?

Comment (aterga) [reply setup.sh:73]: I thought for some releases there's a `nns-dapp_local.wasm` file and for others it's called `nns-dapp.wasm`. Is this not the case?

Comment (mraszyk) [reply setup.sh:73]: nns-dapp.wasm is always available and some releases also contain nns-dapp_local.wasm, but now the WASM is supposed to be environment agnostic and thus nns-dapp.wasm should be ok to use

**Comment by aterga** [comment]
Hi @mraszyk, the basic scenario passed, but the FE test failed. Could this be a brittle test issue?

**Pull request merged**